peers.go

```diff
@@ -549,7 +552,7 @@ func decodePeer(dec *gob.Decoder) (ps peerSummary, connSummaries []connectionSum
 	return
 }
 
-func makeConnsMap(peer *Peer, connSummaries []connectionSummary, byName map[PeerName]*Peer) map[PeerName]Connection {
+func makeConnsMap(peer *Peer, connSummaries []connectionSummary, byName map[PeerName]*Peer, logger *log.Logger) map[PeerName]Connection {
```

Comment:
logger is no longer used here. And hence I think you can get rid of it from `Peer` altogether.